vars/serveRPMs.groovy

```diff
@@ -0,0 +1,40 @@
+#!/usr/bin/groovy
+
+def call(ctx, buildName) {
+  ctx.openshift.withCluster() {
+    def imageStreamRepo = ctx.openshift.selector("is/${buildName}").object().status.dockerImageRepository
+    def output = ctx.openshift.raw("run",
+      "rpm-repo-${buildName}",
+      "--labels", "build=${buildName},app=rpm-repo-${buildName}",
+      "--image", "${imageStreamRepo}:rpms",
+      "--expose", "--port", "8080",
+      "--output", "json", "--dry-run",
+      "--", "/bin/bash", "-c", "'cd /srv/repo && python -m SimpleHTTPServer 8080'"
+    ).out
+    output = new groovy.json.JsonSlurperClassic().parseText(output)
+    for(item in output.items) {
+      if(item.kind == "DeploymentConfig") {
+        item.spec.template.spec.containers[0]["livenessProbe"] = genericProbe()
+        item.spec.template.spec.containers[0]["readinessProbe"] = genericProbe()
+      }
+    }
+    ctx.openshift.apply(groovy.json.JsonOutput.prettyPrint(groovy.json.JsonOutput.toJson(output)))
```

Comment:
In this case the prettyPrint'ed deploymentConfig won't be logged anywhere, we should log it explicitly

Reply:
do we care that it's not? it will get dumped by the artifact gather right

Reply:
I guess we don't, you're right